skimage/filters/thresholding.py

```diff
@@ -1027,8 +1027,9 @@ def apply_hysteresis_threshold(image, low, high):
 
 
 def threshold_multiotsu(image, classes=3, bins=255):
-    """Generates multiple thresholds for an input image.
-    Based on the Multi-Otsu approach by Liao, Chen and Chung.
+    """Generates multiple thresholds for an input image, chosen maximizing the
```

Comment:
Maybe "chosen maximizing" was poorly written. Does it make any sense in this context?

Reply:
"chosen to maximize" is probably more common

Reply:
Done.